android/assets/jsons/translations/Dutch.properties

```diff
@@ -1786,33 +1786,33 @@ Deity = Godheid
 
 Ancient era = Oude tijdperk
  # Requires translation!
-Warrior = 
+Warrior = Strijder
 
-Classical era = Klasieke Eeuw
+Classical era = Klassieke tijdperk
  # Requires translation!
-Spearman = 
+Spearman = Speerman
 
 Medieval era = Middeleeuwen
 
 Renaissance era = Renaissance
  # Requires translation!
-Pikeman = 
+Pikeman = Piekenier
 
 Industrial era = Industrieele
  # Requires translation!
-Musketman = 
+Musketman = Musketier
```

Comment:
Volgens mij niet helemaal, 'Musketier' lijkt me eerder de vertaling van de Franse unique unit 'musketeer'. Misschien eerder iets als 'musketman', als in een man die een musket draagt?